src/ucp/wireup/wireup_cm.c

```diff
@@ -1013,6 +1013,9 @@ ucp_ep_cm_server_create_connected(ucp_worker_h worker, unsigned ep_init_flags,
     ucs_status_t status;
     char client_addr_str[UCS_SOCKADDR_STRING_LEN];
 
+    ep_init_flags |= ucp_cm_ep_init_flags(worker,
+                                          UCP_EP_PARAM_FIELD_CONN_REQUEST);
```

Comment:
UCP_EP_INIT_CM_WIREUP_SERVER | UCP_EP_INIT_CM_PHASE
ucp_cm_ep_init_flags is not logically right to use since it's based on API params

Reply:
fixed